packages/gatsby-theme-docs/src/components/child-sections-nav.js

```diff
@@ -1,33 +1,114 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
-import { designSystem, ContentNotifications } from '@commercetools-docs/ui-kit';
+import {
+  designSystem,
+  ContentNotifications,
+  createStyledIcon,
+} from '@commercetools-docs/ui-kit';
+import SecondaryIconButton from '@commercetools-uikit/secondary-icon-button';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
+import { CloseIcon } from '@commercetools-uikit/icons';
 import { usePageData } from '../hooks/use-page-data';
+import { SearchSvgIcon } from '../icons';
+
+const SearchIcon = createStyledIcon(SearchSvgIcon);
+
+const SearchBoxContainer = styled.div`
+  border: 1px solid ${designSystem.colors.light.borderSecondary};
+  border-bottom: none;
+  padding: ${designSystem.dimensions.spacings.m};
```

Comment:
Forgot the bottom spacing change: 

```suggestion
  padding: ${designSystem.dimensions.spacings.m} ${designSystem.dimensions.spacings.m} ${designSystem.dimensions.spacings.s} ${designSystem.dimensions.spacings.m};
```